### PR TITLE
refactor(style): #2735 switch to new themes structure

### PR DESCRIPTION
**Pull request by francisl**
https://github.com/kids-first/kf-portal-ui/issues/2735

### CODE REVIEW

File: src/style/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Styles and themes
+
+## Strategy
+
+1. Use .less to customize colors and ant design component
```
Comment (evans-g-crsj) [inline src/style/README.md:5]: componentS

File: src/style/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+2. Export to scss to be use for the main theme and custom components
```
Comment (evans-g-crsj) [inline src/style/README.md:6]: Export to scss for the main theme and custom components

File: src/style/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+3. Use css module and scss per components
```
Comment (evans-g-crsj) [inline src/style/README.md:7]: component (no "s")

File: src/style/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+4. Use scss exported variable for props and inline styling (should be avoid when possible)
```
Comment (evans-g-crsj) [inline src/style/README.md:8]: avoided

File: src/style/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+Naming convention for colors variable should follow ant design theme implementation
```
Comment (evans-g-crsj) [inline src/style/README.md:10]: color variableS

**Review by evans-g-crsj — APPROVED** [review]
see comments

**Review by adipaul1981 — APPROVED** [review]

**Pull request merged**